openmls/src/extensions/mod.rs

```diff
@@ -502,4 +510,28 @@ mod test {
             }
         }
     }
+
+    #[test]
+    fn ensure_ordering() {
+        // Create two extensions with different extension types.
+        let x = Extension::ApplicationId(ApplicationIdExtension::new(b"Test"));
```

Comment:
Always use variable names with a meaning. Here, what type of extension for example.
(I know I didn't complain about the ones ebove, but I wanted to get it in ☺️)

Reply:
I prefer to keep this as is in this particular case. The content of the extension is not relevant and I wanted to have short names with the same length so that the `candidates` "tables" can be read better. I added a short comment.

Reply:
As discussed privately, I've renamed it to `ext_{x, y, z}`.